Tasks/HelmDeployV0/task.loc.json

```diff
@@ -14,7 +14,7 @@
   "version": {
     "Major": 0,
     "Minor": 172,
-    "Patch": 0
+    "Patch": 1
```

Comment:
This patch number might conflict

Reply:
Why exactly? The current task version in master is 172.0

Reply:
Got it. Because of the other unmerged PR.